src/topology/vector_bundle.lean

```diff
@@ -0,0 +1,221 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Nicolò Cavalleri.
+-/
+
+import topology.topological_fiber_bundle
+import analysis.calculus.deriv
+import linear_algebra.dual
+
+/-!
+# Topological vector bundles
+
+In this file we define topological vector bundles.
+
+Let `B` be the base space. In our formalism, a topological vector bundle is by definition the type
+`bundle.total_space E` where `E : B → Type*` is a function associating to
+`x : B` the fiber over `x`. This type `bundle.total_space E` is just a type synonym for
+`Σ (x : B), E x`, with the interest that one can put another topology than on `Σ (x : B), E x`
+which has the disjoint union topology.
+
+To have a topological vector bundle structure on `bundle.total_space E`,
+one should addtionally have the following data:
+
+* `F` should be a topological vector space over a field `𝕜`;
+* There should be a topology on `bundle.total_space E`, for which the projection to `E` is
+a topological fiber bundle with fiber `F` (in particular, each fiber `E x` is homeomorphic to `F`);
```

Comment:
Should this say ``for which the projection to `B` is...``?

Reply:
Yes it should

Reply:
```suggestion
* There should be a topology on `bundle.total_space E`, for which the projection to `B` is
```